mm/Kconfig

```diff
@@ -3,6 +3,23 @@
 # see the file kconfig-language.txt in the NuttX tools repository.
 #
 
+choice
+	prompt "Build heap manager"
+	default MM_DEFAULT_MANAGER
+
+config MM_DEFAULT_MANAGER
+	bool "Default heap manager"
+	---help---
+		NuttX original memory manager strategy.
+
+config MM_CUSTOMIZE_MANAGER
+	bool "Customized heap manager"
+	---help---
+		Customized memory manger policy. The build will failed
```

Comment:
```suggestion
		Customized memory manger policy. The build will fail
```

Reply:
hi，when select "MM_CUSTOMIZE_MANAGER", then the user should develop the customize mm first.

please check my patch "mm: add heap policy and anta logic to support mm debug on sim platform #3093",
this is my customize mm policy used under sim platform for sanitizers debug tool.